packages/server/modules/comments/tests/comments.graph.spec.js

```diff
@@ -855,7 +855,7 @@ describe('Graphql @comments', () => {
             [archiveMyComment, true],
             [archiveOthersComment, true],
             [editMyComment, true],
-            [editOthersComment, true],
+            [editOthersComment, false],
```

Comment:
doesn't make sense that anyone other than the comment owner should be able to edit it. otherwise you can have project/workspace owners impersonating others without a trace